Langs/1034.lang

```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=Spanish - Spain
+Translator=Konata400
+Version=2.2.5.0
+[Strings]
+Donate=Donar
+Check for Updates=Buscar actualizaciones
+Your Windows 11 Compatibility Results are Below=A continuación se muestran los resultados de compatibilidad con Windows 11
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Obten ahora WhyNotWin11 via https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=¡Resultados basados ​​en requisitos actualmente conocidos!
+Translation by=Traducción por
+Architecture (CPU + OS)=Arquitectura (CPU + SO)
+Boot Method=Método de arranque
+CPU Compatibility=Compatibilidad CPU
+CPU Core Count=Recuento núcleos CPU
+CPU Frequency=Frecuencia CPU 
+DirectX + WDDM2=DirectX + WDDM2
+Disk Partition Type=Tipo partición disco
+RAM Installed=RAM instalada
+Secure Boot=Secure Boot
+Storage Available=Almacenamiento disponible
+TPM Version=Versión TPM
+Checking...=Comprobando...
+64 Bit CPU=64 Bit CPU
+64 Bit OS=64 Bit SO
+32 Bit CPU=32 Bit CPU
+32 Bit OS=32 Bit SO
+Unable to Determine=Incapaz de determinar
+Unable to Check List=Incapaz de comprobar listado
+Error Accessing List=Error acceder listado
+Not Currently Listed as Compatible=Actualmente no figura como compatible
+Listed as Compatible=Listado como compatible
+Cores=Nucleos
+Threads=Hilos
+GPT Detected=GPT Detectado
+GPT Not Detected=GPT No Detectado
+Enabled=Activado
+Supported=Soportado
+Not Supported=No Soportado
+Disabled / Not Detected=Desactivado / no detectado
+Drive(s) Meet Requirements=La unidad/es cumplen con los requisitos
+TPM Missing / Disabled=Falta TPM / Desactivado
+Detected=Detectado
+DirectX 12 and WDDM 3=DirectX 12 y WDDM 3
+DirectX 12 and WDDM 2=DirectX 12 y WDDM 2
+No DirectX 12, but WDDM2=Sin DirectX 12, pero WDDM2
+DirectX 12, but no WDDM2=DirectX 12, pero sin WDDM2
+No DirectX 12 or WDDM2=No DirectX 12 o WDDM2
+Test Build?=Copilacion Test?
```

Comment:
I think here's also a typo.

Reply:
No DirectX 12, but WDDM2=Sin DirectX 12, pero con WDDM2
DirectX 12, but no WDDM2=DirectX 12, pero sin WDDM2
No DirectX 12 or WDDM2=Sin DirectX 12 o WDDM2
Test Build?=¿Test construcción?

Reply:
You can change your file directly here https://github.com/Konata400/WhyNotWin11/blob/patch-1/Langs/1034.lang

Reply:
Updated